src/command_modules/azure-cli-resource/azure/cli/command_modules/resource/custom.py

```diff
@@ -470,16 +470,15 @@ def export_group_as_template(
     options = ','.join(export_options) if export_options else None
 
     result = rcf.resource_groups.export_template(resource_group_name, ['*'], options=options)
+
+    print(json.dumps(result.template, indent=2))
     # pylint: disable=no-member
     # On error, server still returns 200, with details in the error attribute
     if result.error:
         error = result.error
-        if (hasattr(error, 'details') and error.details and
-                hasattr(error.details[0], 'message')):
-            error = error.details[0].message
-        raise CLIError(error)
-
-    print(json.dumps(result.template, indent=2))
+        logger.warning(result.error.message)
```

Comment:
Hope it helps... So you have verified that `result.error` indeed have the `message` field, right?

Reply:
I can add that validation back in. I'm not sure how non-uniform the errors might be from arm. Thanks for the backup!

Reply:
Done